modules/accredited_representative_portal/app/sidekiq/accredited_representative_portal/power_of_attorney_form_submission_job.rb

```diff
@@ -15,12 +15,17 @@ def perform(poa_form_submission_id)
       @id = poa_form_submission_id
       service = BenefitsClaims::Service.new(poa_form_submission.power_of_attorney_request.claimant.icn)
       @response = service.get_2122_submission(poa_form_submission.service_id)
+      status = (non_error_response? ? :succeeded : :failed)
       poa_form_submission.update(
-        status: (non_error_response? ? :succeeded : :failed),
+        status:,
         service_response: response.to_json,
         status_updated_at: DateTime.current,
         error_message: error_data.to_json
       )
+
+      processing_time = (poa_form_submission.status_updated_at - poa_form_submission.created_at) * 1000
+      StatsD.distribution('ar.poa.submission', processing_time, tags: ["status:#{status}"])
```

Comment:
Should be `ar.poa.submission.duration`. No need for tags.

Reply:
There is some tiny risk someone would change the value and start reporting the metric differently. Possibly test cases can be minimally modified to protect against this. Not too concerned about a regression like that...